meta/run.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
```

Comment:
May I suggest to increase the version required in `meta/package.yaml`? This way, it will only use one version -- note that there really is no need to go with 4.2.6.